### PR TITLE
#15 add the dont show again message to the tutorial

### PR DESCRIPTION
**Pull request by Salam-Dalloul**
- for the first time that user opens the app this tutorial will be displayed, and if clicked on dont show again, then there is a cookie is stored in cookies storage and this tutorial will not appear again 
Screenshot 2023-08-08 at 11 36 21 AM

- if user clicked on tutorial from the hamburger menu, the tutorial will be displayed but without the dont show again checkbox
Screenshot 2023-08-08 at 11 36 03 AM

### CODE REVIEW

File: applications/neuroscan/frontend/src/components/Header.jsx [hunk 1]
```diff
@@ -175,10 +176,17 @@ const Header = (props) => {
...
-      }).start();
+      });
+      intro.start();
     }
   };
 
+  const handleTutorial = () => {
+    setAnchorEl(null);
+    handleViewTutorial();
+    window.document.cookie = 'introjs-dontShowAgain=true; expires=Thu, 18 Dec 2013 12:00:00 UTC';
```
Comment (zsinnema) [inline applications/neuroscan/frontend/src/components/Header.jsx:187]: @Salam-Dalloul Imho this is not needed

**Review by zsinnema — CHANGES_REQUESTED** [review]
@Salam-Dalloul also when I select the don't show and then I try to open the tutorial via the hamburger menu it doesnt show the tutorial, can you have a look? thanks

File: applications/neuroscan/frontend/src/components/Header.jsx (hunk 1 above)
Comment (Salam-Dalloul) [reply applications/neuroscan/frontend/src/components/Header.jsx:187]: @zsinnema yes this was the issue i have talked about on slack, that this cookie needs to be removed when clicking on the hamburger menu but it wan not working with any of the solutions i have tried, and this one of them

File: applications/neuroscan/frontend/src/components/Header.jsx [hunk 2]
```diff
@@ -246,6 +254,10 @@ const Header = (props) => {
     </Popover>
   );
 
+  useEffect(() => {
+    handleViewTutorial();
```
[the remark this answers is not in the document]
Comment (Salam-Dalloul) [reply applications/neuroscan/frontend/src/components/Header.jsx:258]: @zsinnema  as discussed in the meeting this tutorial has to be open directly when the user opens the app until he click on dont show again

**Review by zsinnema — APPROVED** [review]

**Pull request merged**